src/main/java/pl/tomaszdziurko/jvm_bloggers/view/admin/AbstractAdminPage.java

```diff
@@ -32,6 +38,8 @@ private void initializeLeftPanel() {
             MailingPage.class));
         add(new BookmarkablePageLink<AdminDashboardPage>("moderationLink",
```

Comment:
Generics should have the same class as Page they are bound to, so `ModerationPage` here and `BlogsPage` below.